roles/foreman_devel/tasks/check_forks_of_repos.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
What is this intended to do? It doesn't log and it looks like there's no further related actions to it.

Reply:
I'm not sure, what do you mean ? pls more info

Reply:
I think @ekohl is asking what is the intent of this set of tasks? To bomb out if forks don't exist and warn the user?

Reply:
Yes, @ehelms  is right. It is very important that it is testing before the main installation. From my experience, the installation process in not idempotent.

Reply:
@BlackSmith then we should look into why it's not idempotent.

Reply:
@ekol I fully agree, but It is now  over my abilities. :(

Reply:
@BlackSmith if you have specific examples I could have a look.

Reply:
@ekohl What I remember, one of my problem was missing optional or extras repo in RHEL7. Next one is missing of some plugin or katello forks.  The problem with puppet-katello_devel (fixed in pull request 145), problem with IP 169.254.169.254, ...
I'm not sure which one was fatal. Because after few bad tries with continuing, I started to kill old machines and I tried it again with new one.
It is true that before your help (with katello_repositories_environment: staging), we used a little different scenario/playbook. There was for example sometimes problem with "npm install || node ./script/npm_install_plugins.js".  (Maybe you remember https://github.com/BlackSmith/forklift/blob/master/playbooks/full-lunar-devel.yml). Maybe many of this problems are stupid and they are caused only with bad environment, attributes, etc. But it is true, this project has no good documentation and many things are implicite.

Reply:
You're right and we should work on that. It's always been a case where someone solved their own problems. Within the open source community I don't think people really use RHEL7 but opt for CentOS7 which is always easier to use. I tend to ask these kind of questions to get some sense of how it's used and the reasoning so I can try to aim at a unified structure.